imsv-docs-docusaurus/openapi/endpoints/cards/models/card.yaml

```diff
@@ -49,6 +51,14 @@ properties:
   fundingSourceIds:
     type: array
     description: List of Funding Source IDs linked to the card
+  cardProgramId:
+    type: string
+    description: ID of card program linked to the card
+    example: 91ad6fea3b52ca58d60d7fd310f789ec
+  region:
```

Comment:
actually, should it be a `regionCode`? according to this https://immersve.slack.com/archives/C068B0PC5N1/p1726706329395909